LabApiUtilities/build.gradle

```diff
@@ -0,0 +1,70 @@
+plugins {
+    id 'java-library'
+    id 'com.microsoft.identity.buildsystem' version '0.1.0'
+    id 'com.github.gmazzo.buildconfig' version '3.0.0'
+}
+
+def labClientSecret = "" // will be blank unless specified by developer
+
+if (project.hasProperty("labSecret")) {
+    labClientSecret = labSecret
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_7
+    targetCompatibility = JavaVersion.VERSION_1_7
+}
+
+buildSystem {
```

Comment:
Since desugaring is Android specific.  I wonder if we should be updating the plugin to have awareness of both Android and pure Java gradle project types.

Reply:
yes, it already has awareness of both Android and Java gradle projects but it needs to be made more clear and explicit...I will work on making that improvement this week